src/foundry/foundry.js/ray.d.ts

```diff
@@ -34,25 +66,35 @@ declare class Ray {
   slope: number;
 
   /**
-   * The normalized angle of the ray in radians on the range (-PI, PI)
+   * The cached angle, computed lazily in Ray#angle
+   * @internal
+   */
+  protected _angle: number;
```

Comment:
```suggestion
  protected _angle: number | undefined;
```

Also default value `undefined` should be documented.